backend/amazon-prime-now.js

```diff
@@ -2,25 +2,12 @@ const cron = require("node-cron");
 const express = require("express");
 const axios = require("axios");
 const cheerio = require("cheerio");
-const sgMail = require("@sendgrid/mail");
+const sendEmail = require('./utils/send-email');
 
 const merchantId = "A3L2WCBX4NBSPG"; //needs to be checked and changed every time there is a new cart
-// const merchantId = "A3L2WCBX4NBSPG"; // leaving this here since it always returns true and is useful for testing
-const ref = "pn_sc_ptc_bwr";
-const url = `https://primenow.amazon.co.uk/checkout/enter-checkout?merchantId=${merchantId}&ref=${ref}`;
```

Comment:
this cannot be taken out, it is referenced below in line 22 of your new file or line 35 of the original